bbc/simorgh
# fix font order

Resolves JIRA 1679

Overall changes
======
Added "Times New Roman" before Arial as our first backup font for reith qalam. Our current back upfont is too different from the main font, leading to bad content layout shift scores.

Code changes
======

- add `"Times New Roman"` on line 29 in `fontFamilies.ts`
- updated unit tests in `components/Text/index.test.tsx`
- fixed miss matching tests in `components/Paragraph/index.test.tsx`

Testing
======
**relevant unit tests:**
1. run `npx jest src/app/components/Paragraph/index.test.tsx -u`

**manual testing**
1. go to any service that uses reith qalam ([arabic for example](https://www.bbc.com/arabic/business-65050940))
2. go to any article 
3. inspect a paragraph
4. go to its inline style and under 'font-family' and remove `''BBC Reith Qalam"`
5. the paragraph should now fallback to times new roman

Helpful Links
======

[content layout shift](https://web.dev/cls/)

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
